src/index.html

```diff
@@ -4,10 +4,50 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <!-- <h1>Product cards</h1> -->
+
+    <article class="product-card" data-qa="card">
+      <section class="product-card__container">
+
+        <img
+          class="product-card__image"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+        >
+        <p class="product-card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+
+        <p class="product-card__code">Product code: 195434</p>
+
+        <section class="product-card__rating">
+          <ul class="rating-stars">
+            <li class="rating-stars__star"></li>
+            <li class="rating-stars__star"></li>
+            <li class="rating-stars__star"></li>
+            <li class="rating-stars__star"></li>
+            <li class="rating-stars__star"></li>
+          </ul>
+          <p>Reviews: 5</p>
+        </section>
+
+        <section class="price">
+          <p class="price__name">Price:</p>
+          <p class="price__amount">$2,199</p>
+        </section>
+
+        <button
+          class="button-buy"
+          type="submit"
+          data-qa="hover"
+        >BUY</button>
```

Comment:
```suggestion
        >
          BUY
        </button>
```
Consider fixing indentations accordingly to the code style guide kit